BootstrapKit/ModerationPolicy.md

```diff
@@ -0,0 +1,14 @@
+## Working Group Moderation Policy Template
+
+Every Working Group must have a Moderation Policy. By default,
```

Comment:
I liked the phrasing you used for the CoC:

> Every Working Group must have a Code of Conduct. By default, all repositories under the Node.js GitHub Organization fall under the [Node.js Code of Conduct][]

Would it make sense to reuse that wording here? Make it known that one is required, even if it's just a link to the Node.js one?